upgrade_analysis/compare.py

```diff
@@ -0,0 +1,484 @@
+# Copyright 2011-2015 Therp BV <https://therp.nl>
+# Copyright 2015-2016 Opener B.V. <https://opener.am>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# flake8: noqa: C901
+
+#####################################################################
+#   library providing a function to analyse two progressive database
```

Comment:
I've reviewed this file, not in depth. However I see that this is not very much related to a single specific odoo version. Wouldn't `openupgradelib` be a better place to store all this code?

Reply:
Hi. I see the ``openugpradelib`` project a s a set of functions  to make upgrade. That file is not for upgrade but for analysis. so I think it's a good place.

Anyway, such changes can be done in another PR. The objective of this one is just to migrate / rename / move the ``openupgrade_records`` module.